src/MongoDB.Analyzer/Core/Settings/MongoDBAnalyzerSettings.cs

```diff
@@ -25,13 +25,23 @@ internal enum LinqVersion
     V3
 }
 
+[JsonConverter(typeof(StringEnumConverter))]
+internal enum JsonAnalyzerVerbosity
+{
+    None,
+    Medium,
+    All
+}
+
 internal record MongoDBAnalyzerSettings(
     [DefaultValue(false)] bool OutputDriverVersion = false,
     [DefaultValue(false)] bool OutputInternalExceptions = false,
     [DefaultValue(false)] bool OutputInternalLogsToFile = false,
     [DefaultValue(null)] string LogFileName = null,
     [DefaultValue(true)] bool SendTelemetry = true,
     [DefaultValue(null)] LinqVersion? DefaultLinqVersion = null,
+    [DefaultValue(JsonAnalyzerVerbosity.Medium)] JsonAnalyzerVerbosity JsonAnalyzerVerbosity = JsonAnalyzerVerbosity.All,
```

Comment:
We also need tests for both settings.

Reply:
In PocoVerbosity.cs, Included tests for Medium and None